packages/pds/src/lexicon/lexicons.ts

```diff
@@ -10531,6 +10531,51 @@ export const schemaDict = {
       },
     },
   },
+  ToolsOzoneServerGetConfig: {
+    lexicon: 1,
+    id: 'tools.ozone.server.getConfig',
+    defs: {
+      main: {
+        type: 'query',
+        description: "Get details about ozone's server configuration.",
+        output: {
+          encoding: 'application/json',
+          schema: {
+            type: 'object',
+            properties: {
+              appview: {
+                type: 'ref',
+                ref: 'lex:tools.ozone.server.getConfig#serviceConfig',
+              },
+              pds: {
+                type: 'ref',
+                ref: 'lex:tools.ozone.server.getConfig#serviceConfig',
+              },
+              blobDivert: {
+                type: 'ref',
+                ref: 'lex:tools.ozone.server.getConfig#serviceConfig',
+              },
+              viewerRole: {
+                type: 'string',
+              },
+            },
+          },
+        },
+      },
+      serviceConfig: {
+        type: 'object',
+        properties: {
+          configured: {
+            type: 'boolean',
+          },
+          url: {
+            type: 'string',
+            format: 'uri',
+          },
+        },
```

Comment:
What is the purpose of `configured`?  For example, would it ever make sense to serve `{ url, configured: false }`?

Reply:
A related thing to look at is whether any fields should be required.  Right now all of these are in play, for example, and I think they may be interpreted the same way:
```ts
appview: undefined
appview: {}
appview: { configured: false }
appview: { configured: false, url }
```

Reply:
I guess we can infer from the presence of `url` if the service is configured or not which makes the `configured` prop redundant.